### PR TITLE
chinese helloworld doc

### PR DESCRIPTION
**Pull request by sunny0826**

### CODE REVIEW

File: examples/zh/helloWorld.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+[overlays]: ../../docs/glossary.md#overlay
+[patch]: ../../docs/glossary.md#patch
+[variant]: ../../docs/glossary.md#variant
+[variants]: ../../docs/glossary.md#variant
+
+# Demo: hello world with variants
+
+步骤：
+
+ 1. 拉取已经存在的的 [base] 配置。
```
Comment (Liujingfang1) [inline examples/zh/helloWorld.md]: How about
```
下载base配置
```

File: examples/zh/helloWorld.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+ 1. 进行定制。
+ 1. 基于定制后的 base 新建2个不同的 [overlays] (_staging_ 和 _production_)。
```
Comment (Liujingfang1) [inline examples/zh/helloWorld.md]: change 1 to 3

File: examples/zh/helloWorld.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+
+或者：
+
+> ```
+> DEMO_HOME=~/hello
+> ```
+
+## 创建 base
+
+让我们开始运行 [hello] 服务。
```
Comment (Liujingfang1) [inline examples/zh/helloWorld.md]: maybe remove this sentence 
```
让我们开始运行 [hello] 服务。
```

File: examples/zh/helloWorld.md [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+### The Base Kustomization
+
+`base` 目录中包含一个 [kustomization] 文件：
+
+<!-- @showKustomization @test -->
+```
+more $BASE/kustomization.yaml
+```
+
+（可选）在 base 上运行 `kustomize` 将自定义 resources 输出到 `stdout` ：
```
Comment (Liujingfang1) [inline examples/zh/helloWorld.md]: This sentence is confusing
```
在 base 上运行 `kustomize` 将自定义 resources 输出到 `stdout`
```
How about
```
在 base 目录上运行 `kustomize` 将定制过的 resources 打印到标准输出
```
I couldn't find a better translation for `customized`.

File: examples/zh/helloWorld.md [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+
+<!-- @buildBase @test -->
+```
+kustomize build $BASE
+```
+
+### 定制 base
+
+定制 _app label_ 并应用于所有的 resources ：
```
Comment (Liujingfang1) [inline examples/zh/helloWorld.md]:
```
为所有的 resources 添加共同的 _app label_
```

File: examples/zh/helloWorld.md [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+kustomize build $BASE | grep -C 3 app:
+```
+
+## 创建 Overlays
+
+创建包含 _staging_ 和 _production_ 的 [overlay]：
+
+ * _Staging_ 包含生产环境中无法应用的带有风险的功能。
+ * _Production_ 包含更多的副本数。
+ * 来自这些集群 [variants] 的问候将与来自其他集群的不同。
```
Comment (Liujingfang1) [inline examples/zh/helloWorld.md]: 问候 --> 问候消息

File: examples/zh/helloWorld.md [hunk 7]
```diff
@@ -0,0 +1,303 @@
...
+<!-- @overlayDirectories @test -->
+```
+OVERLAYS=$DEMO_HOME/overlays
+mkdir -p $OVERLAYS/staging
+mkdir -p $OVERLAYS/production
+```
+
+#### Staging Kustomization
+
+在 `staging` 目录中创建一个 kustomization 文件，用来定义一个新的 name 前缀和一些不同的 labels 。
```
Comment (Liujingfang1) [inline examples/zh/helloWorld.md]: name 前缀 --> 名称前缀

File: examples/zh/helloWorld.md [hunk 8]
```diff
@@ -0,0 +1,303 @@
...
+  name: the-map
+data:
+  altGreeting: "Have a pineapple!"
+  enableRisky: "true"
+EOF
+```
+
+#### Production Kustomization
+
+在 `production` 目录中创建一个 kustomization 文件，用来定义一个新的 name 前缀和 labels 。
```
Comment (Liujingfang1) [inline examples/zh/helloWorld.md]: name 前缀--> 名称前缀

**Comment by Liujingfang1** [comment]
This is some doc I wrote last year https://zhuanlan.zhihu.com/p/38424955

**Comment by sunny0826** [comment]
> This is some doc I wrote last year https://zhuanlan.zhihu.com/p/38424955

awesome

**Pull request merged**